code/modules/robotics/bot/guardbot.dm

```diff
@@ -97,7 +97,7 @@
 	layer = 5.0 //TODO LAYER
 	density = 0
 	anchored = 0
-	req_access = list(access_heads)
+	req_access = list(access_heads, access_tox, access_tox_storage, access_research, access_chemistry)
```

Comment:
Guardbuddies are one of the few things that the RD has that regular scientists don't. I don't think that flattening the department further is good

Reply:
Done. Only heads can unlock the Buddy, and the DeceptionCheck only looks to see if the person spawned as the RD, or is wearing an RD uniform. If RDs don't like this, they can wear something else and space their dumb outfit, since I think the only other source of one of those uniforms is through a chameleon jumpsuit, and I'm pretty sure that doesn't work here.